app/controllers/cashbacks_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CashbacksController < ApplicationController
+  before_action :authenticate_admin!
+
+  def new
+    @cashback = Cashback.new
+  end
+
+  def create
+    @cashback = Cashback.new(cashback_params)
+    @cashback.admin = current_admin
+    if @cashback.save
+      redirect_to products_path, notice: t('cashback_created')
+    else
+      flash.now[:alert] = t('cashback_not_created')
+      render 'new'
+    end
+  end
+
+  private
+
+  def cashback_params
+    params.require(:cashback).permit(:start_date, :end_date, :percentual)
+  end
+end
```

Comment:
Aqui faltou uma linha em branco no final do arquivo(rubocop).
```suggestion
end

```